src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -129,7 +138,114 @@ static bool mpuSpi6000InitDone = false;
 
 bool mpu6000SpiGyroRead(int16_t *gyroADC);
 bool mpu6000SpiAccRead(int16_t *gyroADC);
-void checkMPU6000Interrupt(bool *gyroIsUpdated);
+void checkMPU6000DataReady(bool *mpuDataReadyPtr);
+
+static bool mpuDataReady;
+static const mpu6000Config_t *mpu6000Config = NULL;
+
+void EXTI3_IRQHandler(void)
```

Comment:
By the way this will only work for interrupts on the CPU pin3 (cc3d) 
I couldn´t get it to work otherwise and so i followed @ledvinap suggestion.

We really could use his IO overhaul :)